src/main/java/frc/robot/RobotContainer.java

```diff
@@ -86,6 +100,15 @@ private void configureBindings() {
     }
   }
 
+  // Register commands for auton
+  public void registerCommands() {
+    HashMap<String, Command> autonCommands = new HashMap<String, Command>();
+
+    autonCommands.put("Shoot", autonTest());
+
+    NamedCommands.registerCommands(autonCommands);
```

Comment:
```suggestion
    NamedCommands.registerCommands(Map.ofEntries(entry("Shoot", autonTest())));
```
Fine as is, just FYI that Map.ofEntries() is a nifty way to make a quick Map literal object. 
Just need to add to imports: 
`import static java.util.Map.entry;`

Reply:
It's nice, but not really readable with long maps like we'll be using for auton